crates/holochain/src/core/ribosome/host_fn/accept_countersigning_preflight_request.rs

```diff
@@ -965,15 +965,15 @@ pub mod wasm_test {
 
     #[tokio::test(flavor = "multi_thread")]
     #[cfg(feature = "slow_tests")]
-    #[cfg_attr(target_os = "macos", ignore = "flaky")]
+    #[ignore = "flaky"]
     async fn enzymatic_session_fail() {
         holochain_trace::test_run().ok();
 
         let (dna_file, _, _) =
             SweetDnaFile::unique_from_test_wasms(vec![TestWasm::CounterSigning]).await;
 
         let mut conductors =
-            SweetConductorBatch::from_config_rendezvous(3, SweetConductorConfig::rendezvous())
+            SweetConductorBatch::from_standard_config(3)
```

Comment:
This was a wrong change. The test expects to be able to do partial reveals so there should not be a bootstrap server available.